### ISSUE TITLE
Trying to figure out the RSSI value

### ISSUE THREAD

**Issue by l0rddarkf0rce** [ewenchou/bluetooth-proximity#14]
I am stumbled by the RSSI value that I get from my Android phone...  Phone is sitting right next to the scanner computer and it always returns 0.  I move away about 10 feet and I still get a 0. What is the -10, 10 tuple supposed to represent?  I was reading https://iotandelectronics.wordpress.com/2016/10/07/how-to-calculate-distance-from-the-rssi-value-of-the-ble-beacon/ and it gives the following as the equation to estimate distance

`Distance = 10 ^ ((Measured Power – RSSI)/(10 * N))`

Another question...

Thanks in advance for any help.

**Comment by ewenchou** [comment]
I saw this happen before, when I kept getting zero values. I was not able to pinpoint what the exact cause was. However, the following two things helped work around it:
1) Reboot the device that's running the Bluetooth adapter (in my case, this was my Raspberry Pi)
2) Unplug other USB devices on Raspberry Pi. I think the USB Bluetooth adapter I was using didn't get enough power. After unplugging the other USB peripherals, I didn't get the zero value problem any more.

The -10, 10 is just a range that I used. When I was testing, I moved my cell phone around my living room and upstairs to see the min/max of the number I was getting. I found that the desired distance I wanted between my phone and raspberry pi was somewhere between -10 and 10.
